installing/installing_ocp/install-central-config-options-ocp.adoc

```diff
@@ -1,13 +1,13 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="install-central-config-options-ocp"]
-= Optional - Configuring Central configuration options for RHACS using the Operator
+= Configuring Central configuration options for RHACS using the Operator
 :context: install-central-config-options-ocp
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 [role="_abstract"]
-This topic provides information about optional configuration options that you can configure using the Operator.
+You can configure optional settings for Central during installation by using the Operator.
```

Comment:
As you know, I'm not a native speaker. But, at first glance, I'm not sure if this is highlighting the correct aspect.

As I understand it, "using the operator" is not some kind of solution for the problem of configuring "optional settings for Central during installation" (not sure if I am reading this correctly -- is that what this sentence suggests?).

It's more like: there are several installation methods, one of them being the operator-based method and this section is supposed to explain how to configure central when using this particular method.

I think the word "by" in "*by* using the Operator" indicates to me that there's this slight difference in meaning here. I guess I wouldn't have stumbled over this if it was "*when* using the Operator" instead(?).

Reply:
That is a good point and I think you're right. It got flagged by our Style guide linter because of a style rule (https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u); however, I should have looked at the context before changing it, because as you said, the context after changing it makes it incorrect.